### PR TITLE
bump to v0.9.9

### PR DESCRIPTION
**Pull request by GopherJ**
Todo in another PR:

switch from `xcm::v0` to `xcm::latest`

### CODE REVIEW

**Comment by xlc** [comment]
Thanks for the RP. Please run `cargo fmt` to format the code.

File: tokens/Cargo.toml [hunk 1]
```diff
@@ -1,43 +1,40 @@
 [package]
-name = "orml-tokens"
+authors     = ["Laminar Developers <hello@laminar.one>"]
```
Comment (shaunxw) [inline tokens/Cargo.toml:2]: Please revert all your format change in `toml` files and keep the previous format and orders.

Comment (shaunxw) [reply tokens/Cargo.toml]: "*and orders*"

File: tokens/Cargo.toml [hunk 2]
```diff
@@ -1,43 +1,40 @@
...
-std = [
-	"serde",
-	"codec/std",
-	"sp-runtime/std",
-	"sp-std/std",
-	"frame-support/std",
-	"frame-system/std",
-	"orml-traits/std",
-]
 runtime-benchmarks = [
```
Comment (xlc) [inline tokens/Cargo.toml:27]: why changing this?

File: tokens/Cargo.toml (hunk 1 above)
Comment (GopherJ) [reply tokens/Cargo.toml]: ok I'll change it

File: tokens/Cargo.toml [hunk 3]
```diff
@@ -1,43 +1,40 @@
 [package]
-name = "orml-tokens"
+authors = ["Laminar Developers <hello@laminar.one>"]
```
Comment (xlc) [inline tokens/Cargo.toml:2]: why changing this?

File: benchmarking/src/lib.rs [hunk 4]
```diff
@@ -600,7 +600,7 @@ macro_rules! benchmark_backend {
 // Every variant must implement [`BenchmarkingSetup`].
 //
 // ```nocompile
-// 
+//
```
Comment (xlc) [inline benchmarking/src/lib.rs:603]: I think you are not using the same rustfmt version with our CI and causing this difference. Please manually revert this change

**Review by xlc — CHANGES_REQUESTED** [review]

File: tokens/Cargo.toml (hunk 2 above)
Comment (GopherJ) [reply tokens/Cargo.toml:27]: seems I should switch to nightly

File: tokens/Cargo.toml (hunk 3 above)
Comment (GopherJ) [reply tokens/Cargo.toml:2]: it's automatically done by editor (formatOnSave)

**Comment by GopherJ** [comment]
@xlc @shaunxw how about now?

**Review by shaunxw — CHANGES_REQUESTED** [review]
The orders of dependencies are still not as origin. Please check them carefully and revert.

**Comment by shaunxw** [comment]
Also please stop force push.

**Comment by xlc** [comment]
and make sure it compiles.

**Comment by GopherJ** [comment]
benchmarking & authority still have some issues, the others are ok now

**Comment by GopherJ** [comment]
hi, guys can you help me on this? I've never used `benchmark` & `authority` pallet.

**Comment by GopherJ** [comment]
close due to noactivity

**Pull request closed without merging**